lib/src/main/java/org/team100/lib/motor/drive/Falcon6DriveMotor.java

```diff
@@ -334,28 +332,28 @@ public void reset() {
     private static double frictionFF(double currentMotorRev_S, double desiredMotorRev_S) {
         double direction = Math.signum(desiredMotorRev_S);
         if (currentMotorRev_S < staticFrictionSpeedLimitRev_S) {
-            return staticFrictionFFVolts * direction / saturationVoltage;
+            return staticFrictionFFVolts * direction ;
```

Comment:
by removing saturation voltage you are changing the meaning of all the FF methods to return volts instead of duty cycle.  it does seem like using voltage feedforward would be better, but if you want to do that, you have to also change all the comments and change the motor.setControl() method to use VelocityVoltage instead of VelocityDutyCycle.